packages/next/src/export/routes/app-page.ts

```diff
@@ -128,10 +129,8 @@ export async function exportAppPage(
 
     const html = result.toUnchunkedString()
 
-    const {
-      metadata: { pageData, revalidate = false, postponed, fetchTags },
-    } = result
     const { metadata } = result
+    const { flightData, revalidate = false, postponed, fetchTags } = metadata
```

Comment:
non blocking nit: there's already a concept of metadata in the app router, could we perhaps chose another more explicit name? `prerenderingMetadata`